frontend/templates/product/sections/ProductReviewsSection/index.tsx

```diff
@@ -217,15 +219,41 @@ function ReviewsStats({
 
 type ProductReviewLineItemProps = {
    review: ProductReview;
+   variantSku: string | undefined | null;
 };
-function ProductReviewLineItem({ review }: ProductReviewLineItemProps) {
+function ProductReviewLineItem({
+   review,
+   variantSku,
+}: ProductReviewLineItemProps) {
+   const isAdminUser = useAuthenticatedUser().data?.isAdmin ?? false;
+   const appContext = useAppContext();
+
    return (
       <Box
          py="6"
          borderBottomWidth="1px"
          borderColor="gray.200"
          data-testid="product-review-line-item"
       >
+         {isAdminUser && (
+            <Button
+               variant="link"
+               as={Link}
+               bgColor="transparent"
+               textColor="brand"
+               fontFamily="heading"
+               lineHeight="1.29"
+               fontWeight="semibold"
+               fontSize="14px"
+               color="brand.500"
+               textAlign="center"
+               paddingBottom="10px"
+               href={`${appContext.ifixitOrigin}/User/Reviews/${variantSku}/userid=${review.author?.userid}`}
```

Comment:
Minor: This link will be broken if variantSku is null or undefined. Maybe we should check for it's truthiness before rendering the button. eg `isAdmionUser && variantSku && ( <Button ... /> )`

Reply:
[ace84e3](https://github.com/iFixit/react-commerce/pull/1573/commits/ace84e31b6ee39b038390ffe16f7debe47e72383)
Thanks!